### DISCUSSION TITLE
More information on the campaign map please

### DISCUSSION THREAD

**Discussion by Benzin73** [FakeFishGames/Barotrauma#4946] [Ideas and suggestions]
Hello, i think it would be great if the map had clear indications of where each biome starts & ends. Currently, there are no labels to say wich biome we are in, or clear lines to delimit the different biomes.
My suggestion is to add vertical "frontier" lines across the map, and labels above each of the delimited sections, to indicate where the different biomes are (so we can tell in wich one we are currently at). Maybe even indicate the depth ranges in each biome?
I dint even know what the different biomes are called, or what their depth ranges are, until found an image online explaining this.

While we are talking about the campaign map. It would be very usefull if each outpost, research station, etc. on the map had a Label indicating to wich faction it belongs to, and our affinity with them. So we can make better decisions when (for example) we plan a trip to get supplies or upgrades. (we would choose a place where they "like us" a lot, so we get better prices).
Currently, i find the affinity system good, but not very useful. As its hard to know wich faction "likes me" more (except for the quick glimpse when a mission ends), and even harder to select an outpost on wich to order expensive upgrades or buy supplies.

Thanks!

**Comment by ubertpendragon** [comment]
I agree that the map needs to communicate more information overall.

Also, with the new offer/demand system for shops, we should be able to see what special deals the shops have to offer on the map

![Capture](https://user-images.githubusercontent.com/73229309/106484585-1092a600-64b0-11eb-8741-7555e09814df.JPG)

**Reply by mLuby** [reply]
> we should be able to

Do you mean we already _are_ able to or we _ought_ to be able to? (I haven't seen the new system yet.)

**Comment by Benzin73** [comment]
Another thing that i think would be very useful, is if each outpost on the map had a list of available crew for hire. Many times i been wondering arround, looking for a medic, guard, etc.
